VocaDbWeb/Views/Shared/KnockoutPartials/_CoverArtistMessage.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿<!--
+	Shows a message for cover artists in an original song.
+	Binding context:  KnockoutComputed<dc.ArtistContract[]>
+-->
+<div class="alert" style="display:none" data-bind="visible: coverArtists() && coverArtists().length > 0 && !canHaveOriginalVersion()">
+	<partial name="Partials/Shared/_NotificationIcon" />
+	@ViewRes.EntryCreateStrings.FoundCoverArtist
+	<ul data-bind="foreach: coverArtists">
+		<li>
+			<a data-bind="text: name, attr: { href: app.EntryUrlMapper.details(EntryType.Artist, id) }"></a>

```

Comment:
Sorry, but this should be `app.EntryUrlMapper.details('Artist', id)` because it's not TypeScript.